README.md

```diff
@@ -0,0 +1,92 @@
+# Corpuscule
+
+[![Build Status](https://img.shields.io/travis/corpusculejs/corpuscule/master.svg)](https://travis-ci.org/corpusculejs/corpuscule)
+[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE)
+[![Test Coverage](https://img.shields.io/codecov/c/github/corpusculejs/corpuscule/master.svg)](https://codecov.io/gh/corpusculejs/corpuscule)
+
+Corpuscule is a library set built on top of WebComponents standard. It provides all necessary tools
```

Comment:
Consider using "Web Components" (separately), not "WebComponents" here and below